### DISCUSSION TITLE
今天似乎有问题，播放有困难了。

### DISCUSSION THREAD

**Discussion by yelaoniu** [HerbertHe/iptv-sources#102] [Q&A]
今天似乎有问题，播放有困难了。之前一直相当好用的，估计是地址失效了？同学们有这个问题吗？

**Comment by xj9696** [comment]
挂了有些天了！

**Comment by yelaoniu** [comment]
能修复好吗？习惯用您的了。谢谢！

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by HerbertHe** [comment]
ghproxy.net 被封了，之前更新失败

**Comment by yelaoniu** [comment]
哦！明白，我再想其他办法。谢谢。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by yelaoniu** [comment]
复活了！太好了。谢谢为大家服务！

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
